experiments/build_vars.cfg

```diff
@@ -0,0 +1,17 @@
+# Variables needed and used by the various building and reoprting scripts.
+
+# File extensions to expect for Specification files and Body files
+spec_ext="${SPEC_EXT:-ads}"
+body_ext="${BODY_EXT:-adb}"
+
+# Sane ADA_INCLUDE_PATH and GPR_PROJECT_PATH?
+GNAT2GOTO=`command -v gnat2goto`
+ADA_HOME=`command -v gnat`
+ADA_HOME=$(cd "$(dirname ${ADA_HOME})/.." 2>/dev/null && pwd)
+PLATFORM=`${ADA_HOME}/bin/gcc -dumpmachine`
+DEF_ADA_INCLUDE_PATH="${ADA_HOME}/lib/gcc/${PLATFORM}"
+ADA_GCC_VERSION=`${ADA_HOME}/bin/gcc -dumpversion`
+DEF_ADA_INCLUDE_PATH="${DEF_ADA_INCLUDE_PATH}/${ADA_GCC_VERSION}/rts-native/adainclude:${ADA_HOME}/include"
+export ADA_INCLUDE_PATH="${ADA_INCLUDE_PATH:-$DEF_ADA_INCLUDE_PATH}"
+
+file_name=$(basename "${project_dir}")
```

Comment:
I think `file_name` should go away - it's an artefact of our current list_unsupported.sh script, but we wouldn't expect the users build system to know it needs to generate this special file name...